batch2/Dockerfile.worker

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:18.04
+
+ENV LANG C.UTF-8
+
+RUN apt-get update && \
+  apt-get -y install \
+    python3 python3-pip && \
+  rm -rf /var/lib/apt/lists/*
+
+COPY docker/requirements.txt .
```

Comment:
We might be able to do a little better by just installing what the worker needs, although it might most of it.  This seems like a great start.